src/components/Quote/Quote.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Box,
+  Container,
+  Grid,
+  IconButton,
+  Stack,
+  useTheme,
+} from '@mui/material';
+import { type CommonProps, type Generic } from 'types/components';
+import QuoteText from './QuoteText';
+import { useState } from 'react';
+import SwipeableViews from 'react-swipeable-views';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+
+export interface QuoteProps extends CommonProps, QuoteTextProps {
+  image?: string | Generic;
+  altText?: string;
+  inverse?: boolean;
+  background?: string;
+  size: string;
+  quotes: QuoteTextProps[];
+}
```

Comment:
It seems that image, altText, inverse and size are not used props. Please remove them from this interface.

Reply:
done